### PR TITLE
Fix mwc-list/menu actions

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Fix double actions when using request-selected

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/hui-root.ts [hunk 1]
```diff
@@ -58,6 +58,8 @@ import type { Lovelace } from "./types";
 import "./views/hui-panel-view";
 import type { HUIPanelView } from "./views/hui-panel-view";
 import { HUIView } from "./views/hui-view";
+import { RequestSelectedDetail } from "@material/mwc-list/mwc-list-item";
```
Comment (zsarnett) [inline src/panels/lovelace/hui-root.ts:61]:
```suggestion
import type { RequestSelectedDetail } from "@material/mwc-list/mwc-list-item";
```

Comment (bramkragten) [reply src/panels/lovelace/hui-root.ts:61]: If it is only a type it will be thrown out on compile, but I guess it can't hurt to add it.

Comment (bramkragten) [reply src/panels/lovelace/hui-root.ts]: Oh it was duplicate import ofc 😄

File: src/common/util/handle-request-selected-event.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {
```
Comment (balloob) [inline src/common/util/handle-request-selected-event.ts:1]: Can we move this file to an `common/mwc` ?

**Review by balloob — APPROVED** [review]

**Pull request merged**